### PR TITLE
Add fuzzing targets for Lwan

### PR DESCRIPTION
**Pull request by lpereira**
This PR adds fuzz targets for the Lwan project.  The corpora being used is generated automatically from the test suite.

### CODE REVIEW

File: projects/lwan/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
+
+mkdir -p $WORK/lwan
+cd $WORK/lwan
+
+cmake -GNinja \
+	-DCMAKE_BUILD_TYPE=Debug -DSANITIZER=${SANITIZER} \
```
Comment (Dor1s) [inline projects/lwan/build.sh]: How important the `$SANITIZER` value is? We generally don't recommend to rely on env variables other than the ones specified here: https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#requirements

File: projects/lwan/build.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ninja -v liblwan.a
+
+zip -jr $OUT/request_fuzzer_seed_corpus.zip $SRC/lwan/fuzz/corpus/corpus-request-*
+
+for fuzzer in $SRC/lwan/src/bin/fuzz/*_fuzzer.cc; do
+	executable=$(basename $fuzzer .cc)
+	$CXX $CXXFLAGS -std=c++11 \
+		-Wl,-whole-archive $WORK/lwan/src/lib/liblwan.a -Wl,-no-whole-archive \
+		-I$SRC/lwan/src/lib $fuzzer \
+		-fsanitize=${SANITIZER} \
```
Comment (Dor1s) [inline projects/lwan/build.sh]: This is not needed, the necessary sanitizer flags are in `$CXXFLAGS` already

File: projects/lwan/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM gcr.io/oss-fuzz-base/base-builder
```
Comment (Dor1s) [inline projects/lwan/Dockerfile]: Could you please add the copyrights to this file and build.sh? They are usually created automatically by `infra/helper.py` script, or you can copy it from any other project files.

File: projects/lwan/build.sh (hunk 1 above)
Comment (lpereira) [reply projects/lwan/build.sh]: Indeed, with `-DCMAKE_C_FLAGS`, this is not needed.

File: projects/lwan/build.sh (hunk 2 above)
Comment (lpereira) [reply projects/lwan/build.sh]: Ack.

**Comment by lpereira** [comment]
Thanks for the review.  Force-pushed with the requested changes.

**Pull request merged**